regexes/device/mobiles.yml

```diff
@@ -7141,12 +7232,22 @@ Roku:
   device: 'tv'
   model: 'Digital Video Player'
 
+# Rokit
+Rokit:
+  regex: 'IO Pro'
+  device: 'smartphone'
+  models:
+    - regex: 'IO Pro'
+      model: 'IO Pro'
+
 # Rover
 Rover:
-  regex: 'Rover ([0-9]+)'
+  regex: 'rokit io pro 3d'
```

Comment:
this change does not seem to make any sense, guess you wanted to add that in the detection before...

Reply:
Revert change